src/predicate/isWindow.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, it } from 'vitest';
+import { isWindow } from './isWindow.ts';
+
+describe('isWindow', () => {
+  it('should return "false" if the element is not the `Window` object', () => {
+    expect(isWindow({})).toBeFalsy();
+    expect(isWindow([])).toBeFalsy();
+    expect(isWindow(1)).toBeFalsy();
+    expect(isWindow('')).toBeFalsy();
+  });
+});
```

Comment:
When I tested with both environments of `vitest` set to `jsdom` and `happy-dom`, both environments caused issues with `isWindow` and other separate test files. So, first of all, I only wrote the test code to pass.

Reply:
1. jsdom: https://github.com/toss/es-toolkit/actions/runs/14547617444
    - tests/check-dist.spec.ts
    - src/object/clone.spec.ts 
  
<img width="1322" alt="스크린샷 2025-04-19 오후 6 25 05" src="https://github.com/user-attachments/assets/600bbede-5e02-4f80-a375-62d771744ffb" />

2. happy-dom: https://github.com/toss/es-toolkit/actions/runs/14547701146
    - tests/check-dist.spec.ts
 
<img width="1329" alt="스크린샷 2025-04-19 오후 6 26 36" src="https://github.com/user-attachments/assets/7fbf666a-978d-43b4-b515-ef1aaa652ea1" />

Reply:
If you have any good advice for test code, let us know! 🙏